src/pybind/mgr/crash/module.py

```diff
@@ -3,6 +3,7 @@
 import errno
```

Comment:
commit msg should probably be "pybind/mgr/crash:..." (unlike my "mgr/pybind/crash..." (sigh))

Reply:
We mostly shorthand this to "mgr/<module name>" so it'll be "mgr/crash: "

Reply:
It's annoying to have an undocumented convention but I'm not sure where the heck I'd document it!

Reply:
@jcsp github lets you create pull request templates. adding common prefixes and link to a master list would be an option.

Reply:
The trouble is that it's a commit message thing rather than a PR thing (even though PR text usually comes out of commit text)

Reply:
I was imaging the PR template as being used as a checklist/instructions as its information you interact with right before submitting the PR: "have you (a) ... (b) ... and (c) checked your commit message formats against list X?"